client/components/profilePage/main.jsx

```diff
@@ -1,13 +1,14 @@
 import MyLibrary from './myLibrary.jsx';
 import MyProfile from './myProfile.jsx';
-import React from 'react';
+import React, { useContext } from 'react';
 import css from './profileStyles.css';
+import { AuthContext } from '../../context/authContext.jsx';
```

Comment:
Context looks unused? Assuming it didn't work or something?

Reply:
Was experimenting, we are getting user from props so dont need context.